tests/loss/test_TicoLoss.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import torch
+
+from lightly.loss.tico_loss import TiCoLoss
+torch.manual_seed(0)
```

Comment:
Please add whitespace after imports and move `torch.manual_seed(0)` into each test method.